### DISCUSSION TITLE
Documentation navigation improvement

### DISCUSSION THREAD

**Discussion by Zen0x7** [laravel/framework#52162] [Ideas] [answered]
I know that laravel documentation site has a good search engine but my idea isn't directly about that.

You know that using mouse is a huge waste of time. Having said that, don't how many times i been on the middle of the documentation or at the end and in order to jump to another section of documentation i had to go to google and find the new concept to open the new documentation site, it is clearly something behavioral. 

The point is, and i'm not sure if that exists, but why not implement a shortcut to open the algolia modal?

> When i was working on one of my last job, i had the idea, on a big application (a lot of tables), implement a portal to jump between modules of the application. So you can open with `CTRL + J` and write, `go to` in a input field, so all the system entities (model types) appears, if you select one by clicking or pressing arrows, you can press `enter` and decide if you wanna jump to the `index` or to specific `instance` or even more, to relationships or sections of that object. Of course, that was only jump between instances like a object navigation. 

I'm pretty sure that will be a nice improvement in terms of navigation, in other words, you could avoid jumping to the beggining of the page and manually pressing to the search input to find another section. So we can have a documentation with UI nagivation mechanisms like a Vim professional user. I'm not trying to reinvent the wheel.

**Comment by gbalduzzi** [comment, marked as answer]
`cmd` + `k` opens the Algolia modal for me, I guess on windows `ctrl` + `k` does the same

**Reply by Zen0x7** [reply]
Thanks @gbalduzzi
